semantic_conventions.md

```diff
@@ -55,6 +57,15 @@ The following Span tags combine to model RPCs:
 - `error`: whether the RPC ended in an error
 - `peer.hostname`, `peer.ipv4`, `peer.ipv6`, `peer.port`, `peer.service`: optional tags that describe the RPC peer (often in ways it cannot assess internally)
 
+### Message Oriented Middleware
+
+The following Span tags combine to model Message Oriented Middleware communications:
+
+- `mom.destination` and `mom.type`: as described in the table above
+- `span.kind`: either `"producer"` or `"consumer"`. It is important to provide this tag **at Span start time**, as it may affect internal ID generation.
+- `error`: whether the message exchange ended in an error
```

Comment:
IMO this could be left out... it seems (?) clear to me that `error` can apply basically anywhere.